### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/get-started/install.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#564] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/get-started/install.html

安装 在 [dir] 文件夹内新建 vuepress-theme-hope 项目: tab pnpm tab yarn tab npm 要将 vuepress-theme-hope 作为文档构建器添加到现有项目中，请在项目根目录中运行以下命令: tab pnpm tab yarn tab npm 这里的 [dir] 是一个参数，你需要使用真实的文件夹名...

https://theme-hope.vuejs.press/zh/guide/get-started/install.html

**Comment by meowhuan** [comment]
# centos系统部署出错
> 提示分段错误

**Comment by XiaoYang943** [comment]
测试一下。
